README.md

```diff
@@ -1 +1,66 @@
-# android-developer-fundamentals codelabs V2
+
+# Android App Development
+This repository contains solutions for the Android App Development course
+
+Introduction
+------------
+
+Each of the samples in this repository is associated with a practical
+exercise from the Android App Development course. While each 
+solution stands and runs all alone, they should be utilized along with the course. 
+
+This Android App Development course aims on providing basic knowledge about Android programming concepts and practical knowledge on developing a wide range of apps, using Java. 
+
+Getting Started
+---------------
+
+1. [Install Android Studio](https://developer.android.com/studio/install.html),
+if you don't already have it
+2. Download the samples/solutions
+2. Import the sample into Android Studio
+3. Build and run the sample
+
+Index
+---------------
+
+| S. No        | Name           |
+| ------------- |-------------| 
+|1.1  | [Hello World](https://github.com/yashk2000/android-codelabs/tree/master/1.1) |
+|1.2A  | [Hello Toast](https://github.com/yashk2000/android-codelabs/tree/master/1.2A/HelloToast) |
+|1.2B  | [Layout Editing](https://github.com/yashk2000/android-codelabs/tree/master/1.2B) 
+|1.3  | [Text and scrolling views](https://github.com/yashk2000/android-codelabs/tree/master/1.3) |
+|1.4  | [Helping Yourself](https://github.com/yashk2000/android-codelabs/tree/master/1.4) |
+|||
```

Comment:
Give a blank, and also mention the module name for each different module like you mentioned in the doc.
Looks fine other than that :rocket: